ArgusCore/src/test/java/com/salesforce/dva/argus/entity/InfractionTest.java

```diff
@@ -0,0 +1,110 @@
+package com.salesforce.dva.argus.entity;
```

Comment:
This test should be re-written to extend abstractTest and re-use the entitymanager that's established there.  We can talk about this and discuss exactly how we want it to work.